packages/gatsby/src/commands/build-html.ts

```diff
@@ -101,7 +101,7 @@ class BuildHTMLError extends Error {
   constructor(error: Error) {
     super(error.message)
 
-    Object.keys(error).forEach(key => {
+    Object.getOwnPropertyNames(error).forEach(key => {
```

Comment:
Could you add a comment why this matters?

Reply:
you mean in the code? or just here for you?

Reply:
in code :) Like a comment and probably in the prepare-stack-trace as well.